src/openforms/authentication/contrib/yivi_oidc/api/views.py

```diff
@@ -0,0 +1,30 @@
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import authentication, permissions
+from rest_framework.generics import GenericAPIView
+from rest_framework.mixins import ListModelMixin
+
+from ..models import AttributeGroup
+from .serializers import AttributeGroupSerializer
+
+
+@extend_schema_view(
+    get=extend_schema(summary=_("List available Yivi attribute groups")),
+)
+class AttributeGroupListView(ListModelMixin, GenericAPIView):
```

Comment:
```suggestion
class AttributeGroupListView(generics.ListAPIView):
```
see https://www.django-rest-framework.org/api-guide/generic-views/#listapiview